src/renderer/App.tsx

```diff
@@ -120,6 +123,52 @@ function Sidebar() {
 }
 
 function Topbar() {
+  const wallet = useWallet();
+  const validator = useAppSelector(selectValidatorNetworkState);
+  const { net } = validator;
+
+  useEffect(() => {
+    const airdropIfNeeded = async () => {
+      if (validator.status !== NetStatus.Running) return;
+      let amount = 0;
+
+      // arbitrary -- want to let people top up
+      // on devnet SOL when they can
```

Comment:
these should all be in the settings dialog (in a future release)

Reply:
agreed!